build.rs

```diff
@@ -583,7 +583,8 @@ fn configure_cc(c: &mut cc::Build, target: &Target, c_root_dir: &Path, include_d
         let _ = c.define("NDEBUG", None);
     }
 
-    if (target.arch == X86) && !compiler.is_like_msvc() {
+    let is_msvc_not_clang_cl = compiler.is_like_msvc() && !compiler.is_like_clang_cl();
+    if (target.arch == X86) && !is_msvc_not_clang_cl {
```

Comment:
I suggest instead:
```
if taret_arch = X86 {
    let is_msvc_not_clang_cl = ...';
    if !is_msvc_not_clang_cl { ... }
}
```
This way, there will be no effect for people building for non-x86 targets.

Also, in Cargo.toml, we'd need to increase the cc version to 1.2.8 as `is_like_clang_cl` was added then.